src/main/io/gps.c

```diff
@@ -1126,8 +1648,9 @@ static void parseFieldNmea(gpsDataNmea_t *data, char *str, uint8_t gpsFrame, uin
 
         case FRAME_GGA:        //************* GPGGA FRAME parsing
             switch (idx) {
-    //          case 1:             // Time information
-    //              break;
+                case 1:
+                    data->time = ((uint8_t)(str[5]) * 10 + (uint8_t)(str[7])) * 100;
```

Comment:
`str` seems to be in ascii, so 
```suggestion
                    data->time = ((uint8_t)(str[5] - '0') * 10 + (uint8_t)(str[7] - '0')) * 100;
```

Reply:
Thank you, have fixed it as suggested, will push soon.